docker/tidy/ansible/roles/demisto_server/files/.demisto_bashrc

```diff
@@ -0,0 +1,62 @@
+################################################ General ################################################
+export GOENV_ROOT=$HOME/.anyenv/env/goenv
+export GOPATH=$HOME/dev/go
+export GOROOT=~/.anyenv/envs/goenv/versions/1.16.0
+export PATH=$GOENV_ROOT/bin:$GOPATH/bin:$PATH
+export GOENV_DISABLE_GOPATH=1
+eval "$(goenv init -)"
+
+alias demisto="cd $HOME/dev/demisto"
+
+################################################ Content ################################################
+export CONTENT=$HOME/dev/demisto/content
+export PYTHONPATH="${CONTENT}:${PYTHONPATH}"
+
+################################################ Client ################################################
+export CLIENT=$HOME/dev/demisto/web-client
+alias client-build='cd $CLIENT_FOLDER && npm run build-light'
+alias client-start='cd $CLIENT_FOLDER && npm start light'
+
+################################################ Server ################################################
+export SERVER=$GOPATH/src/github.com/demisto/server
+
+# General
+alias clear-debug='rm -rf $SERVER/debug'
+alias copy-one-time='cp $SERVER/conf/testdata/onetimeconf.json $SERVER/ && chmod +rw $SERVER/onetimeconf.json'
+alias compile-server='cd $SERVER && go mod tidy -v && go install -mod=vendor && go install -mod=vendor'
+
+start-clean-server="$HOME/dev/go/src/github.com/demisto/server/start-clean-server.sh"
+start-server="$GOPATH/bin/server -public ~/dev/demisto/web-client/dist -conf conf_for_systemtests_elasticsearch -insecure -loglevel debug -stdout"
```

Comment:
@Shellyber this is missing `alias`
```suggestion
alias start-clean-server="$HOME/dev/go/src/github.com/demisto/server/start-clean-server.sh"
alias start-server="$GOPATH/bin/server -public ~/dev/demisto/web-client/dist -conf conf_for_systemtests_elasticsearch -insecure -loglevel debug -stdout"
```

Reply:
@yaakovi https://github.com/demisto/dockerfiles/pull/4989